rep-2000.rst

```diff
@@ -600,43 +600,43 @@ Minimum language requirements:
-| Cyclone DDS |                                 0.5.0 (Eusebius)                                     |
-+-------------+----------------+---------------+----------------+----------------+-------------------+
-|  Fast-RTPS  |                                      1.9.3                                           |
-+-------------+----------------+---------------+----------------+----------------+-------------------+
++-------------+-------------------------------------------------+---------------------------+-------------------+
+|             |                        Required Support         |              Recommended Support              |
++-------------+----------------+---------------+----------------+----------------+----------+-------------------+
+| Package     |  Ubuntu Focal  |     MacOS**   |   Windows 10** | Debian Buster  | CentOS 7 | OpenEmbedded Thud |
++=============+================+===============+================+================+==========+===================+
+| CMake       |     3.13.4     |     3.14.4    |     3.14.4     |      3.13.4    |  3.14.6  |       3.12.2      |
++-------------+----------------+---------------+----------------+----------------+----------+-------------------+
+| EmPY        |                                              3.3.2                                              |
++-------------+----------------+---------------+----------------+----------------+----------+-------------------+
+| Gazebo      |     11.0.0     |     11.0.0    |      N/A       |      11.0.0    |             N/A              |
++-------------+----------------+---------------+----------------+----------------+----------+-------------------+
+| Ignition    |             Citadel            |      N/A       |     Citadel    |             N/A              |
++-------------+----------------+---------------+----------------+----------------+----------+-------------------+
+| Ogre        |                                     1.10*                                   |        N/A        |
++-------------+----------------+---------------+----------------+----------------+----------+-------------------+
+| OpenCV      |     4.1.0      |     4.1.0     |     3.4.6*     |      3.2.0     |   N/A    |        3.2.0      |
++-------------+----------------+---------------+----------------+----------------+----------+-------------------+
+| OpenSSL     |     1.1.1c     |     1.0.2r    |     1.0.2r     |      1.1.1c    |  1.0.2k  |1.1.1b / 1.0.2r****|
++-------------+----------------+---------------+----------------+----------------+----------+-------------------+
+| Poco        |     1.9.2      |     1.9.0     |     1.8.0*     |      1.9.0     |  1.8.0*  |        1.9.0      |
++-------------+----------------+---------------+----------------+----------------+----------+-------------------+
+| Python      |     3.8.0      |     3.7.3     |     3.8.0      |      3.7.3     |  3.6.8   |        3.7.2      |
```

Comment:
Just pointing out that Python 3.6.8 conflicts with the minimum Python version mentioned earlier (3.7). I think this is okay since CentOS is listed as a Tier 3 platform. But I think it's very possible that we may break compatibility if we're not running CI regularly for CentOS (this happened with Debian in a past release). My only concern is that if we're planning to ship binaries for CentOS that it's possible we will ship broken Python packages. 

@dirk-thomas thoughts?

Reply:
Hm, that is a bummer. I don't think we want to lower the minimum Python version at this point.

If any Python code needed in a core part this will basically mean CentOS 7 isn't feasible.

Reply:
:(

CentOS 7 has Python 3.3, 3.4 and 3.6. Judging from the amount of work it was getting 3.6 ramped up, I don't think we should expect an update any time soon.

If 3.6.8 isn't good enough, I'll have to drop this until we can get sufficient packages in EPEL 8 and try again then.